resqpy/time_series.py

```diff
@@ -247,6 +247,18 @@ def extend_by_days(self, days):
       duration = TimeDuration(days = days)
       self.extend_by_duration(duration)
 
+   def datetimes(self):
+      """Returns a list of python-datetime objects """
+      datetimes=[]
+        
+      for timestamp in self.timestamps:
+         datetime_timestamp=timestamp
+         if timestamp.endswith('Z'): datetime_timestamp = timestamp[:-1]
+         datetimes.append(dt.datetime.fromisoformat(datetime_timestamp))
+        
+        
+
+      return datetimes
```

Comment:
Perhaps a list comprehension is simpler here?

```suggestion
      return [dt.datetime.fromisoformat(t.rstrip('z')) for t in self.timestamps]
```